### DISCUSSION TITLE
Help to Add Domains And Subdomains To Multi Tenancy

### DISCUSSION THREAD

**Discussion by Sabreena-Hussien** [filamentphp/filament#11476] [Help]
### Package

Panel builder

### Package Version

v3.2

### How can we help you?

I create a system using Filament v3 using the multi tenancy and it work fine.
now the client ask me to add support for domains and subdomains to the app.
I tried this package "Stancl\Tenancy" and didn't work for me, I cant make it work, I also use Custom Tenant Model Called "Comapny", that make it's more difficult form me with this package.

is there any approach can I follow it to achieve this. I am using single database.
Thank you.

**Comment by cweagans** [comment]
https://filamentphp.com/docs/3.x/panels/tenancy#using-a-domain-to-identify-the-tenant
